scientist.js

```diff
@@ -35,8 +35,7 @@ exports.create = function(/* constructor, ...constructorArgs */) {
   var machine;
 
   if (constructor.prototype) {
-    machine = Object.create(constructor.prototype);
-    machine.constructor.apply(machine, constructorArgs);
+    machine = new (Function.prototype.bind.apply(constructor, [null].concat(constructorArgs)));
```

Comment:
No, because the first element given to bind (the context) is overwritten by the `new`. See http://stackoverflow.com/a/35878917